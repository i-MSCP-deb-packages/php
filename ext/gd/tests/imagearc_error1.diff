002+ 40a085f6e7d9b5f3622d69da5549e64f
002- abebb25b5a2813cfbf92f1f24365786a