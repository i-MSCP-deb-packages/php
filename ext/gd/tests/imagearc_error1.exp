Warning: imagearc() expects exactly 8 parameters, 7 given in %s on line %d
abebb25b5a2813cfbf92f1f24365786a