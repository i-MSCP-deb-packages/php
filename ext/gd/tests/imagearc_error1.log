
---- EXPECTED OUTPUT
Warning: imagearc() expects exactly 8 parameters, 7 given in %s on line %d
abebb25b5a2813cfbf92f1f24365786a
---- ACTUAL OUTPUT
Warning: imagearc() expects exactly 8 parameters, 7 given in /home/ondrej/Projects/pkg-php/php-src/ext/gd/tests/imagearc_error1.php on line 8
40a085f6e7d9b5f3622d69da5549e64f
---- FAILED
