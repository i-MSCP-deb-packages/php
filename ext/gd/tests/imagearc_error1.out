Warning: imagearc() expects exactly 8 parameters, 7 given in /home/ondrej/Projects/pkg-php/php-src/ext/gd/tests/imagearc_error1.php on line 8
40a085f6e7d9b5f3622d69da5549e64f